### PR TITLE
fix push block notify bug

### PR DESCRIPTION
**Pull request by morebtcg**
fix push block notify bug

### CODE REVIEW

File: libinitializer/RPCInitializer.cpp [hunk 1]
```diff
@@ -78,14 +78,14 @@ void RPCInitializer::initConfig(boost::property_tree::ptree const& _pt)
             auto blockChain = m_ledgerManager->blockChain(it);
             auto channelRPCServer = std::weak_ptr<dev::ChannelRPCServer>(m_channelRPCServer);
             auto handler = blockChain->onReady([groupID, channelRPCServer](int64_t number) {
-                LOG(TRACE) << "Push block notify: " << (int)groupID << "-" << number;
+                LOG(INFO) << "Push block notify: " << (int)groupID << "-" << number;
```
Comment (wheatli) [inline libinitializer/RPCInitializer.cpp]: use GROUP_ID instead of int

File: libinitializer/RPCInitializer.cpp [hunk 2]
```diff
@@ -78,14 +78,14 @@ void RPCInitializer::initConfig(boost::property_tree::ptree const& _pt)
...
                 auto c = channelRPCServer.lock();
 
                 if (c)
                 {
                     std::string topic =
                         "_block_notify_" + boost::lexical_cast<std::string>((int)groupID);
-                    std::string content = boost::lexical_cast<std::string>(groupID) + "," +
+                    std::string content = boost::lexical_cast<std::string>((int)groupID) + "," +
```
Comment (wheatli) [inline libinitializer/RPCInitializer.cpp]: use GROUP_ID instead of int

File: libinitializer/RPCInitializer.cpp [hunk 3]
```diff
@@ -78,14 +78,14 @@ void RPCInitializer::initConfig(boost::property_tree::ptree const& _pt)
...
             auto channelRPCServer = std::weak_ptr<dev::ChannelRPCServer>(m_channelRPCServer);
             auto handler = blockChain->onReady([groupID, channelRPCServer](int64_t number) {
-                LOG(TRACE) << "Push block notify: " << (int)groupID << "-" << number;
+                LOG(INFO) << "Push block notify: " << (int)groupID << "-" << number;
                 auto c = channelRPCServer.lock();
 
                 if (c)
                 {
                     std::string topic =
                         "_block_notify_" + boost::lexical_cast<std::string>((int)groupID);
```
Comment (wheatli) [inline libinitializer/RPCInitializer.cpp]: use GROUP_ID instead of int

File: libchannelserver/ChannelRPCServer.cpp [hunk 4]
```diff
@@ -846,15 +846,16 @@ std::vector<dev::channel::ChannelSession::Ptr> ChannelRPCServer::getSessionByTop
     std::vector<dev::channel::ChannelSession::Ptr> activedSessions;
 
     std::lock_guard<std::mutex> lock(_sessionMutex);
-    for (auto& it : _sessions)
+    for (auto it : _sessions)
```
Comment (wheatli) [inline libchannelserver/ChannelRPCServer.cpp]: why can not use & ?

File: libinitializer/RPCInitializer.cpp (hunk 1 above)
Comment (cyjseagull) [reply libinitializer/RPCInitializer.cpp]: fixed in: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/527/files#diff-fc72f03b22cdddcf4af335bee13f73e6

File: libinitializer/RPCInitializer.cpp (hunk 2 above)
Comment (cyjseagull) [reply libinitializer/RPCInitializer.cpp]: fixed in: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/527/files#diff-fc72f03b22cdddcf4af335bee13f73e6

File: libinitializer/RPCInitializer.cpp (hunk 3 above)
Comment (cyjseagull) [reply libinitializer/RPCInitializer.cpp]: fixed in: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/527/files#diff-fc72f03b22cdddcf4af335bee13f73e6

File: libchannelserver/ChannelRPCServer.cpp (hunk 4 above)
Comment (cyjseagull) [reply libchannelserver/ChannelRPCServer.cpp]: fixed in: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/527/files#diff-8bf794f0b9e4c11b691b052256f5afa4

**Pull request merged**